README.md

```diff
@@ -48,10 +48,14 @@ export base64_script_content=$(cat init-interfaces.sh|base64 -w 0) && envsubst <
 - Follow the guide to install a bare-metal cluster in the [OpenShift production documentation.
 ](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/installing/installing-on-bare-metal)
 
+### 4. Apply the kubernetes-nmstate NNCP Bond configuration
+- Create bond `oc apply -f add-slb-nncp.yaml`
+- Delete the bond `oc delete nncp --all && oc apply -f del-slb-nncp.yaml`
```

Comment:
Should we list this as a standalone note? Having it here may risk that users would copy copy copy everything without realizing they are removing what they just created.

Reply:
Ahh you mean Put the delete as a note right ? ok, make sense